ghcide/exe/Main.hs

```diff
@@ -73,7 +73,7 @@ main = withTelemetryRecorder $ \telemetryRecorder -> do
     -- stderr recorder just for plugin cli commands
     pluginCliRecorder <-
       cmapWithPrio pretty
-      <$> makeDefaultStderrRecorder (Just [PriorityColumn, DataColumn])
+      <$> makeDefaultStderrRecorder (Just [ThreadIdColumn, PriorityColumn, DataColumn])
```

Comment:
Hmm, this seems like it will affect people using the ghcide executable directly? Well, hopefully that will go away in due course anyway...

Reply:
Yes, I am under the impression ghcide is just using for test purpose now and we are considering removing it?